### PR TITLE
chore: update the version of the github actions used

### PR DESCRIPTION
**Pull request by philasmar**
*Description of changes:*
Update the version of the github actions used to fix the warnings around Node.JS 12

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml [hunk 1]
```diff
@@ -482,13 +503,61 @@ Resources:
...
+        Fn::Sub: cdk-${Qualifier}-permissions-boundary-${AWS::AccountId}-${AWS::Region}
+      Path: /
   CdkBootstrapVersion:
     Type: AWS::SSM::Parameter
     Properties:
       Type: String
       Name:
         Fn::Sub: /cdk-bootstrap/${Qualifier}/version
-      Value: "14"
+      Value: "15"
```
Comment (ashovlin) [inline src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml:560]: Could this be moved out to a separate PR since it may have changes for users?

Or else update this PR's commit message to mention the bootstrap stack changing?

Comment (philasmar) [reply src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml]: Created a new PR https://github.com/aws/aws-dotnet-deploy/pull/751

File: .github/workflows/stale_issues.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ jobs:
     name: Stale issue job
     runs-on: ubuntu-latest
     steps:
-    - uses: aws-actions/stale-issue-cleanup@main
+    - uses: aws-actions/stale-issue-cleanup@v5
```
Comment (ashovlin) [inline .github/workflows/stale_issues.yml:13]: V6 was just released an hour ago 😆 

https://github.com/aws-actions/stale-issue-cleanup/releases/tag/v6

File: src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml (hunk 1 above)
Comment (ashovlin) [reply src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml]: Thanks!

File: .github/workflows/stale_issues.yml (hunk 2 above)
Comment (philasmar) [reply .github/workflows/stale_issues.yml]: Done

**Review by ashovlin — APPROVED** [review]

**Review by 96malhar — APPROVED** [review]
